chpl/chpl-api/src/main/resources/log4j2-xinclude-file-appenders-console.xml

```diff
@@ -105,16 +105,16 @@
             <KeyValuePair key="dd.span_id" value="%X{dd.span_id}" />
         </JsonLayout>
     </Console>
-    <Console name="inheritanceErrorsReportCreatorJobJson" target="SYSTEM_OUT">
+    <Console name="icsErrorsReportCreatorJobJson" target="SYSTEM_OUT">
         <JsonLayout compact="true" eventEol="true" properties="true" stacktraceAsString="true">
-            <KeyValuePair key="service" value="inheritanceErrorsReportCreatorJob" />
+            <KeyValuePair key="service" value="icsErrorsReportCreatorJob" />
```

Comment:
Will changing these names affect the Datadog reports? I don't know if those look for reports by name, or something else

Reply:
On the dockerized system(s) the Datadog reports should just work with the new name. On the other systems I'll need to update the config in Bamboo if we want to be able to see the logs in datadog. Added a checklist item.